gdal/src/main/scala/geotrellis/raster/gdal/GDALUtils.scala

```diff
@@ -41,25 +41,18 @@ object GDALUtils {
       case _ => throw new Exception(s"Could not find equivalent GDALResampleMethod for: $method")
     }
 
-  def dataTypeToCellType(datatype: GDALDataType, noDataValue: Option[Double] = None, typeSizeInBits: => Option[Int] = None, minMaxValues: => Option[(Double, Double)] = None): CellType =
+  def dataTypeToCellType(datatype: GDALDataType, noDataValue: Option[Double] = None, typeSizeInBits: => Option[Int] = None): CellType =
     datatype match {
       case TypeByte =>
         typeSizeInBits match {
           case Some(bits) if bits == 1 => BitCellType
           case _ =>
-            minMaxValues match {
```

Comment:
A useless computation and check; since GDAL stores everything as UByte, it always would be a value from the range `[0; 255)`.

Reply:
❤️